ci/pipelines/deploy-to-test.yml

```diff
@@ -4415,6 +4416,108 @@ jobs:
           image: selfservice-ecr-registry-test/image.tar
           additional_tags: selfservice-ecr-registry-test/tag
 
+  - name: codebuild-cardid
+    plan:
+      - in_parallel:
+        - get: pay-ci
+        - get: cardid-git-release
+          trigger: false # TODO change this when we're done
+          params:
+            submodules: none
+      - task: parse-release-tag
+        file: pay-ci/ci/tasks/parse-release-tag.yml
+        input_mapping:
+          git-release: cardid-git-release
+      - task: update-submodule
+        config:
+          container_limits: {}
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: alpine
+              tag: 3.16
+          inputs:
+            - name: cardid-git-release
+          params:
+            GH_ACCESS_TOKEN: ((github-access-token))
+          outputs:
+            - name: cardid-git-release-with-submodules
```

Comment:
Don't think we need submodules anymore. cardID submodule is required only for building the image which CodeBuild does.

Reply:
I use it to get the `RELEASE_NAME`. Can we get that from the main repo as well?

Reply:
yes. we don't submodule for tags